backend/src/api/fishtanks/index.js

```diff
@@ -0,0 +1,44 @@
+const { Router } = require('express');
+const FishtankController = require('../../controllers/fishtank.js');
+
+const router = new Router();
+
+/**
+ * @api {get} /fishtanks Get fishtanks
+ * @apiName GetFishtanks
+ * @apiGroup Static Pages
+ * @apiDescription This URL displays the current list of fishtanks
+ * @apiSuccessExample {html} Success-Response:
+ *     HTTP/1.1 200 OK
+ *     Fishtanks list:
+ */
+router.get('/', (req, res) => res.send('Fishtanks list:'));
+
+/**
+ * @api {post} /fishtanks Post fishtank
+ * @apiName PostFishtanks
+ * @apiDescription This URL receives new fishtank declarations
```

Comment:
Missing `@apiGroup`, it looks very weird on the documentation: https://ebm2018.github.io/TenuGUI/#api-_home_travis_build_EBM2018_TenuGUI_backend_src_api_fishtanks_index_js